pkg/core/apiversion/apiversion.go

```diff
@@ -0,0 +1,35 @@
+package apiversion
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+)
+
+// MajorVersion is the current major API Version.
+const MajorVersion = 2
+
+// MinorVersion is the current minor API Version.
+const MinorVersion = 0
```

Comment:
So we start from 2.0? I was thinking of following the milestone version (i.e. currently 2.1) for easier tracking.